easybuild/scripts/rpath_wrapper_template.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
so again, why both a bash and python wrapper? Use a purely python one? Forks are costly.